CHANGELOG.md

```diff
@@ -27,8 +27,11 @@
   `Prelude.File` and to provide a 'better' API.
 * `natEnumFromThen` and `natEnumFromTo` have been updated to correctly calculate reverse ranges. Range syntax `[a,b..c]` now can be used again to generate reverse ranges.
 * `divBN` and `modBN` now can only be used for unsigned numbers.
-
 * `return`, which has been an alias for `pure` for many releases, is now deprecated.
+* `InstanceN` is deprecated, use `ImplementationN` instead.
+* `InstanceCtorN` is deprecated, use `ImplementationCtorN` instead.
```

Comment:
I would be nicer here to have a top level not of the renaming with these bullet points being presented as a sublist.